content/applications/general/companies.rst

```diff
@@ -4,89 +4,196 @@
+
+Multiple companies can be set for :guilabel:`Allowed Companies`, and *only one* can be set as the
+:guilabel:`Default Company`.
 
 .. image:: companies/access_rights_multi_companies.png
    :align: center
    :alt: View of an user form emphasizing the multi companies field under the access rights tabs
-         in Odoo
+         in Odoo.
+
+If an administrator has multiple companies activated on the database, and is editing a record, the
+editing occurs on the record's related company.
+
+.. example::
+   If editing a sale order issued under `JS Store US`, while working on the `JS Store Belgium`
+   environment, the changes are applied under `JS Store US` (the company from which the sale order
+   was issued).
+
+When creating a record, the company taken into account is:
+
+- The current company selected in the company selector, in the upper-right hand of the screen (the
+  one that is highlighted/active)
 
-| If a user has multiple companies *activated* on his database, and he is **editing** a record,
-  the editing happens on the record's related company.
-| Example: if editing a sale order issued under JS Store US while working on the JS Store Belgium
-  environment, the changes are applied under JS Store US (the company from which the sale order
-  was issued).
-| When **creating** a record, the company taken into account is:
+**OR**
 
-- The current company (the one active) or,
-- No company is set (on products and contacts’ forms for example) or,
-- The company set is the one linked to the document (the same as if a record is being edited).
+- No company is set (because none is set on the product and contact forms, for example)
 
-Documents’ format
-=================
+**OR**
 
-To set documents' formats according to each company, *activate* and *select* the respective one and,
-under *Settings*, click on *Configure Document Layout*.
+- The company set is the company linked to the document (the same as if a record is being edited)
+
+Document format
+===============
+
+To set document formats according to each company, *activate* and *select* the respective company,
+and, under the :menuselection:`Settings app`, click on :guilabel:`Configure Document Layout` and
+edit the information as needed.
 
 .. image:: companies/document_layout.png
    :align: center
-   :alt: View of the settings page emphasizing the document layout field in Odoo
+   :alt: View of the settings page emphasizing the document layout field in Odoo.
+
+:guilabel:`Company Details` can be edited on the document layout. By default, this field is
+populated from the company information listed, when navigating here: :menuselection:`Settings app
+--> Companies section --> Manage Companies`, and select a company from the list.
 
 Inter-Company Transactions
 ==========================
 
-First, make sure each one of your companies is properly set in relation to:
+**First**, activate the :ref:`developer mode <developer-mode>`. Then, make sure each one of the
```

Comment:
The bold **First** formatting seems a little strange to me, but I understand it is calling attention that debug mode **must** be activated in order to see the _Inter-Company Transactions_ tab. 

Maybe there is a better way to structure this notice just before navigating to the _Inter-Company Transactions_ tab since debug mode doesnt necessarily need to be enabled to configure the Chart of Accounts, Taxes, etc...?
```suggestion
First, activate the :ref:`developer mode <developer-mode>`. Then, make sure each one of the
```